docs/en/Configuring-Alluxio-with-NFS.md

```diff
@@ -14,15 +14,22 @@ First, the Alluxio binaries must be on your machine. You can either
 [compile Alluxio](Building-Alluxio-Master-Branch.html), or
 [download the binaries locally](Running-Alluxio-Locally.html).
 
-Then, if you haven't already done so, create your configuration file from the template:
+Then, if you haven't already done so, create your configuration file with `bootstrapConf` command.
+For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_HOSTNAME` should be set to `localhost`
```

Comment:
I mean the value in the command. If you think this is not clear, we should update all occurrences of this pattern as it is used multiple places across our docs